privileges-grantor/README.md

```diff
@@ -44,6 +42,54 @@ module "apiary-privileges-grantor" {
 
 ```
 
+The apiary-privileges-grantor lambda can be found in the public [maven repository](https://mvnrepository.com/artifact/com.expediagroup.apiary/apiary-privileges-grantor-lambda).
+The jars can be downloaded from the link provided above and uploaded to S3 via terraform as follows:
+
+```
+variable "pg_lambda_version" {
+  description = "Version of the Privilege Grantor Lambda."
+  type        = "string"
+  default     = "4.2.0"
+}
+
+data "aws_s3_bucket" "apiary-extensions" {
+  bucket = "pg-s3-bucket"
+}
+
+resource "null_resource" "apiary-privileges-grantor-jar" {
```

Comment:
I wouldn't do this point to a place on s3. either the shared maven repos if possible otherwise just copy the jar manually once. You really don't want do to this every time you apply.

Reply:
Missed the bit where this was a example in the readme :) yeah all good